intro-to-debugging/intro-to-debugging.md

```diff
@@ -59,8 +59,6 @@ Now that we have a general set of questions and order to debugging, here is a ge
 6. Document what you tried and what the results were... Probably on a piece of paper or a separate file.
 7. Repeat this process until the bug is fixed!
 
-True or false: It is okay to come up with theories about the bug and to test & experiment, as long as you document what you tried and what the results are in your notes
```

Comment:
I see what you mean about this sentence, the "true or false" part is kind of funny. I think it's ok to remove it completely, but I also feel like it could remain in the doc if we changed it to be like: 

"Note: It is okay to come up with theories about the bug and to test & experiment, as long as you document what you tried and what the results are in your notes"

Reply:
I lean towards removing it. It seems like a summary of steps 4-7 directly above. Does that feel okay @yangashley ?

Reply:
Removing all together sounds good to me too!